### PR TITLE
Linters and first sections

### PR DESCRIPTION
**Pull request by carreraprogrammer**
In this branch:

- I added Linters
- I added the navbar and header and customized them in accord with the "mobile first" parameters 
- I added the vectors into the header

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300;1,700&family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="styles.css">
+      <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      <header>
+        <nav class="navbar">
+            <p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p>
```
Comment (Ismailco) [inline index.html]: - [x] The logo should be on the left of the top bar and has a link (a tag), kindly wrap it on an `<a>` tag so it's clickable

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        </nav>
+        <section class="welcome-section">
+            <h1>Hey There<span>.</span><br> I'm Carrera</h1>
+            <h2>I'm a Full Stack Developer</h2>
+            <p>I can print your dreams in a web page. If you like what you see and have a project you need coded, don't hestiate to contact me</p>
+            <div class="social-icons">
+                <i class="fa-brands fa-github-alt"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-twitter"></i>
+            </div>
```
Comment (Ismailco) [inline index.html]: - [x] The list of icons should use a `<ul>` tag to group the icons, and `<li>` per icon, then each icon should be a link by using `<a>` tag
e.g: `<li><a href="url">icon</a></li>`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @carreraprogrammer, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @carreraprogrammer 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**